e2e/data-factory/persistence/adapters/api.ts

```
@@ -0,0 +1,99 @@
+            await this.delete(entityType, id);
+        }
+    }
+    
+    async findById<T>(entityType: string, id: string): Promise<T | null> {
+        try {
+            const response = await this.agent
+                .get(`${this.endpoint}/${id}`)
+                .expect(200);
+            
+            return this.extractResponse(response.body);
+        } catch (error: any) {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)